### PR TITLE
Fix html build for dmd

### PR DESCRIPTION
**Pull request by andralex**
This fixes the dmd side of the html build. Pull for dlang.org coming shortly.

### CODE REVIEW

File: src/lexer.d [hunk 1]
```diff
@@ -1360,7 +1360,7 @@ public:
     /**************************************
      * Lex delimited strings:
      *      q"(foo(xxx))"   // "foo(xxx)"
-     *      q"[foo(]"       // "foo("
+     *      q"[foo$(LPAREN)]"       // "foo$(LPAREN)"
```
Comment (yebblies) [inline src/lexer.d]: Can you use ` instead?

File: src/parse.d [hunk 2]
```diff
@@ -1438,7 +1438,7 @@ public:
      * Parse template parameter list.
      * Input:
      *      flag    0: parsing "( list )"
-     *              1: parsing non-empty "list )"
+     *              1: parsing non-empty "list $(RPAREN)"
```
Comment (ZombineDev) [inline src/parse.d]: Would it work if you change it to `` `list )` ``?
Ditto for the places below.

Comment (andralex) [inline src/parse.d]: It works but the doc build issues a warning.

Comment (andralex) [inline src/parse.d]: Oh you mean make it code. Let me try.

Comment (andralex) [inline src/parse.d]: No, didn't work.

**Comment by WalterBright** [comment]
Auto-merge toggled on

**Comment by andralex** [comment]
Fatal Error: Out of memory

**Comment by WalterBright** [comment]
yeah, that seems unrelated. Wonder what's up with that.

**Comment by WalterBright** [comment]
I guess it was a heisenbug!

**Pull request merged**